tsconfig.json

```diff
@@ -13,7 +13,8 @@
         // These settings are required for TypeScript project references
         "declaration": true,
         "declarationMap": true,
-        "sourceMap": true
+        "sourceMap": true,
+        "useUnknownInCatchVariables": false
```

Comment:
This is a new thing in [TS 4.4](https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables) where the value in a catch clause is assumed to be `unknown` and you have to assert that the type is actually `Error`. Since you can reject promises with any value in JS 🤦 But this causes a bunch of errors in our code and since we only throw Errors I think it's fine to turn this check off.